mlir/include/mlir/Analysis/Presburger/Utils.h

```diff
@@ -292,6 +294,53 @@ std::vector<Fraction> multiplyPolynomials(ArrayRef<Fraction> a,
 
 bool isRangeZero(ArrayRef<Fraction> arr);
 
+/// An example to print .12, 3.4, 56.7 where preAlign = ".", minSpacing = 1,
+/// and (`,~) indicates (PreIndent, PostIndent) respectively:
```

Comment:
Its fine to just put spaces and not have special characters here.

Reply:
Removed special characters and used space.